packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/ActionButton.tsx

```
@@ -20,14 +17,15 @@ export function ActionButton({
   amount,
   token,
   totalValue,
+  ...props
 }: IActionButtonProps) {
   const actionText = tradeType === 'buy' ? 'Buy' : 'Sell';
   const numericAmount = parseFloat(amount);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)